app/controllers/actions/IdentifierAction.scala

```diff
@@ -69,19 +68,13 @@ class AuthenticatedIdentifierAction @Inject() (
           redirectToLogin
         case _: InsufficientEnrolments =>
           redirectToEoriComponent
-        case _: InsufficientConfidenceLevel | _: UnsupportedAuthProvider | _: UnsupportedAffinityGroup |
```

Comment:
you cant reach this branch from user journey perspective as we dont request that data from user so this branch cant be triggered